core/app/models/spree/stock_transfer.rb

```diff
@@ -100,6 +104,18 @@ def close(closed_by)
 
     private
 
+    def ensure_transfer_items_valid
```

Comment:
This does return true/false, so I think a name like: `transfer_items_valid?` would be better.